what-is-software-development/study-like-a-programmer.md

```diff
@@ -0,0 +1,340 @@
+
+A maybe-helpful analogy: If our jobs were to paint color-by-numbers, we could probably maximize productivity and output by working long hours day after day. If our jobs are to create inspiring artwork, **practicing self-care (rest, etc.) is an essential part of our work.**
+
+### Take Care of Your Brain
+
+Visualize your brain as a tool that needs care and maintenance to be most effective. That means when something in your program isn't working and your brain is experiencing stress, your brain is probably not going to be very productive until you've given it a break and taken care to restore your brain to a less heightened state. 
+
+### !challenge
+
+* type: multiple-choice
+* id: a7753324-aa1f-4858-85d4-a0d1956fb36b
+* title: Programming is Creative Work
+
+##### !question
+
+Based on the information above, which of the following is **_not_** an appropriate metaphor?
+
+Programmers are like...
+
+##### !end-question
+
+##### !options
+
+* Painters who are given materials and constraints, and must use creativity to make something
+* Computers who must get the right answer every time
+* Athletes who must practice and take care of themselves in order to improve
+* Managers who must keep track of projects and health at the same time
+
+##### !end-options
+
+##### !answer
+
+* Computers who must get the right answer every time
+
+##### !end-answer
+
+### !end-challenge
+
+## Studying Like a Programmer
+
+Studying programming, especially if you have not been in the classroom in a while, is really challenging! If you don't have a grasp on what study habits work best for you, it will take time and experimentation to find out what's most effective.
+
+The best programmers improve by focusing on these study skills:
+
+1. Question-Asking Skills
+1. Review Skills
+1. Perserverence & Self-Care Skills
+
+### Your Question-Asking Skills
+
+#### What You Know
+
+First, let's convince ourselves that there are different levels of knowledge and learning. Knowing and understanding something has more nuance than "I get it" or "I don't get it."
+
+Here's a common model of "knowing things" that we can reference:
+
+Level | Depth of knowledge | Description
+--- | --- | ---
+1 | I know what I know | I know what I've been taught
+2 | I don't know what I don't know | I'm learning enough to know there's more to learn
+3 | I know what I don't know | I've learned enough to know what I've learned vs. what more to learn
+4 | I don't know what I know | I don't even realize how much I've learned
+
+Levels 1 and 2 should describe where everyone is right now with programming. In fact, we should spend most of our times as learners in level 1 and 2.
+
+Level 3 is the destination that we want all of our careers to point to.
+
+Level 4 describes an experienced programmer. It is _not_ our destination.
```

Comment:
I'd add:
> (though you'll probably experience this at some point in your post-Ada career).

Reply:
I appreciate the suggestion! I feel pretty strongly to let this statement be more "simple"/straightforward-- by the time a student even needs to consider level 4, they'll understand that... they're there. :P ? Not sure if my logic follows, but I'd like to not expound on Level 4.

Reply:
I'm resolving this to help me feel like I'm closing this thread, not to shut down any conversation you might want to have. If you want, please continue on Slack! @ace-n <- tagging for visibility, tho i'm sure github has sent you emails about this lol